app/models/saved_claim/education_benefits/va_10203.rb

```diff
@@ -9,29 +9,35 @@ class Submit10203EVSSError < StandardError
 
   # rubocop:disable Metrics/CyclomaticComplexity
   def after_submit(user)
-    create_stem_automated_decision(user) if user.present?
+    @user = user
+    if @user.present?
+      @gi_bill_status = get_gi_bill_status
+      create_stem_automated_decision
+    end
+
 
     email_sent(false)
     return unless FeatureFlipper.send_email?
 
     StemApplicantConfirmationMailer.build(self, nil).deliver_now
 
-    if user.present?
+    if @user.present?
       education_benefits_claim.education_stem_automated_decision.update(confirmation_email_sent_at: Time.zone.now)
-      authorized = user.authorize(:evss, :access?)
+      authorized = @user.authorize(:evss, :access?)
 
-      EducationForm::SendSchoolCertifyingOfficialsEmail.perform_async(user.uuid, id) if authorized
+      EducationForm::SendSchoolCertifyingOfficialsEmail.perform_async(id ,less_than_six_months?, get_facility_code) if authorized
```

Comment:
https://github.com/mperham/sidekiq/wiki/Best-Practices#1-make-your-job-parameters-small-and-simple